script/vsts/lint.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+  - job: Lint
+    timeoutInMinutes: 10
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+      - template: templates/cache.yml
+        parameters:
+          OS: linux
```

Comment:
The cache template restores more than needed (top level deps and apm folder)! Installing script dependencies is faster.

Reply:
Yeah, that's what I'm trying to measure now. It looks like it's 30 seconds per cache hit, and 1:30 for straight install. Now the CI invocation is fixed the straight install may be faster, but is it possible to only restore the cache we care about?

Reply:
Ah, it's too much hassle. I've removed caching.